main.py

```diff
@@ -0,0 +1,148 @@
+import csv
+import logging
+
+
+class Account:
+    def __init__(self, name):
+        self.name = name
+        self.balance = 0
+        self.transactions = []
+
+    def add_money(self, amount):
+        self.balance += amount
+
+    def substract_money(self, amount):
+        self.balance -= amount
+
+    def add_transaction(self, date, narrative, amount, paid): # if paid is True -> person sent amount

```

Comment:
If the comments describe the function parameters, then use `docstrings` please: https://peps.python.org/pep-0257/